### ISSUE TITLE
Cannot use keyboard only navigation to select specific pages in pagination options

### ISSUE THREAD

**Issue by JamesDuncanFlindersUni** [sajari/website-search-integration#14]
Hi,

When navigating the website integration search interface, a user who is restricted to keyboard-only navigation is unable to select or interact with numbered pagination options, however, they can interact with the prev and next pagination buttons.

For users that fit this profile, it is generally required that clickable elements, especially those that are not inputs or 'a' tags with 'href' values, have a tabindex value, or can be navigated to without a mouse.

tabindex="0" on each .sj-paginator__page-number would probably be sufficient.

Please see gif below demonstrating issue:

![sajari_keyboard_navigation_pagination_issue](https://user-images.githubusercontent.com/47310558/53707748-2d676e80-3e80-11e9-8195-7139c6410b45.gif)

**Comment by benhinchley** [comment]
Hey @JamesDuncanFlindersUni This is by design, when testing we found that screen readers when implementing WAI-ARIA, support we chose to only allow the ability to get to the prev/next buttons to reduce the number of keyboard presses to get to the next page. We find it very rare that anyone clicks a specific page number, so we made the most common path the easiest. If there is a valid argument against this, I'm happy to look into it again.

**Comment by JamesDuncanFlindersUni** [comment]
@benhinchley All good, so long as there was a business requirement behind it, I am fine with just the next and prev options being available.
